style.css

```diff
@@ -0,0 +1,74 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    }
+  
+    section{
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      min-height: 100vh;
+      background: #042104; 
+      animation: animateBg 10s linear infinite;
+    }
+    
+    @keyframes animateBg
+    {
+     0%
+     {
+        filter: hue-rotate(0deg);
+     }   
+     100%
+     {
+        filter: hue-rotate(360deg);
+     }
+    }
+    
+    section .loader{
+        position: relative;
+        width: 120px;
+        height: 120px;
+    }
+    
+    section .loader span{
+        position:absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        transform: rotate(calc(18deg * var(--i)));
+    }
```

Comment:
What does calc function do?

Reply:
it calculate the things like frequency angle length etc